src/relay_pic/nbproject/private/configurations.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
nbproject/private is just configuration specific to your machine. It shouldn't be tracked in git.